test/rules/require_trailing_comma.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+class RequireTrailingCommaExample {
+  RequireTrailingCommaExample.constructor1(Object param1, Object param2);
+
+  RequireTrailingCommaExample.constructor2(
```

Comment:
The only function type I see missing is operators. Since trailing commas don't make sense on an operator call (a BinaryExpression, UnaryExpression, or IndexExpression), I would only suggest adding a test for an operator declaration, like `void operator [](LongType longParameterName) {`

Reply:
Added test case for operator declaration.